### DISCUSSION TITLE
Can I submit mupltiple entries?

### DISCUSSION THREAD

**Discussion by MonstraG** [SebLague/Chess-Challenge#52] [Q&A]
I have several ideas for wonky chess engines, can I submit them? If yes, how would I go about it - google forms requires login and creating several accounts feels wrong.

**Comment by Shibulski** [comment]
Let them play against (one always be EvilBot) eachother and submit the best one.

**Comment by lucasmeneghin** [comment]
I would also like if multiple submissions are allowed

**Comment by TronFortyTwo** [comment]
I am thinking about multiple design approaches. Even if one may be stronger in the game it could be interesting to submit both of them. At the end the competition is not just about making the strongest bot

**Comment by draeg82** [comment]
I think a hard 2 bot submission limit per entrant would be fair. It would allow for a conventional not as well as allow freedom for some unconventional bots to be seen. Either that make a serious tournament Vs fun for all version.

**Comment by SebLague** [comment]
Hi, I've just updated the rules on this. I'd request that you submit a maximum of 2 entries, and to only submit a second entry if you feel it is significantly different from your first bot. Note, this will require signing in with a second account.

**Reply by AngelFireLA** [reply]
It would be easier for the 2 submission if instead of forcing the Google account's mail you asked the email in a field where we can type the email ourselves.
